packages/@react-aria/color/src/useColorSlider.ts

```diff
@@ -61,6 +62,43 @@ export function useColorSlider(props: ColorSliderAriaOptions, state: ColorSlider
     inputRef
   }, state);
 
+  let {minValue, maxValue, step, pageSize} = state.value.getChannelRange(channel);
+  let {keyboardProps} = useKeyboard({
+    onKeyDown(e) {
+      // these are the cases that useMove or useSlider don't handle
+      if (!/^(PageUp|PageDown|Home|End)$/.test(e.key)) {
+        e.continuePropagation();
+        return;
+      }
+      // same handling as useMove, stopPropagation to prevent useSlider from handling the event as well.
+      e.preventDefault();
+      let channelValue = state.value.getChannelValue(channel);
+      let pageStep = Math.max(pageSize, step);
+      let newValue = channelValue;
+      switch (e.key) {
+        case 'PageUp':
+          newValue = channelValue + pageStep > maxValue ? maxValue : snapValueToStep(channelValue + pageStep, minValue, maxValue, pageStep);
```

Comment:
useColorSliderState should have increment/decrement thumb functions https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/color/src/useColorSliderState.ts
see
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/slider/src/useSliderState.ts#L125

We should try to match this logic to useSlider as best we can

Reply:
Just a heads up, but I think @majornista was running into some issues w/ using the `setThumbValue` not updating the color value [here](https://github.com/adobe/react-spectrum/pull/2570#discussion_r813450449), need to double check myself